WcaOnRails/app/views/committee_positions/_committee_position_form.html.erb

```diff
@@ -0,0 +1,16 @@
+  <%= simple_form_for [@committee, @position], url: url, html: { class: 'form-horizontal are-you-sure no-submit-on-enter' }, wrapper: :horizontal_form,
+    wrapper_mappings: {
+      check_boxes: :horizontal_radio_and_checkboxes,
+      radio_buttons: :horizontal_radio_and_checkboxes,
+      file: :horizontal_file_input,
+      boolean: :horizontal_boolean
+    }  do |f| %>
+    <%= render 'shared/error_messages', object: f.object %>
+
+    <%= f.input :name %>
+    <%= f.input :description, hint: t('.description_html', md: t('common.hints.supports_md_html')) %>
+    <%= f.input :team_leader %>
+    <%= link_to icon("undo", "Back to #{@committee.name}"), committee_path(@committee), class: "btn btn-primary" %>
```

Comment:
@viroulep what are your thoughts here? We use both translated hint that make sense at least because of reusing `'supports_md_html'`, and untranslated `"Back to #{@committee.name}"` that makes sense as well because it's not a public page. I think we don't want to end up with pages mixed of multiple languages.